docs/units-and-global-variables.rst

```diff
@@ -167,6 +167,10 @@ For more information, see the section on :ref:`address`.
     to make safe Ether transfers, always check the return value of ``send``, use ``transfer`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+.. note::
+   Using ``delegatecall`` without a library requires the storage of the calling and called contracts to align 
```

Comment:
Perhaps better not to mention the word `library` at all and instead use something like `If storage variables are accessed via a low-level delegatecall, the storage locations...`

Reply:
Makes sense!